src/event_object_movement.c

```diff
@@ -7701,7 +7703,7 @@ static void GetGroundEffectFlags_JumpLanding(struct ObjectEvent *objEvent, u32 *
     }
 }
 
-static u8 ObjectEventCheckForReflectiveSurface(struct ObjectEvent *objEvent)
+static u8 GetObjEventReflectionType(struct ObjectEvent *objEvent)
```

Comment:
The `return 0;` and `result != 0` in this function should use `REFL_TYPE_NONE`. Also while we're here let's move the definition of `RETURN_REFLECTION_TYPE_AT` to above the function definition, it's confusing to define it in the middle of the code.

Could you also elaborate on the name change? For me `GetObjEventReflectionType` implies the reflection type is part of the object event, when what it seems to be doing is getting the reflection type of metatiles around the object event

Reply:
Ah, yeah I wasn't sure whether to move `RETURN_REFLECTION_TYPE_AT`, it did seem like a weird place for it. Nice catch with the 0s too, I think I saw them and then forgot to change them.

As for the name change, my approach stemmed from being misled by `Check`-- to me, I expected it to return either TRUE or FALSE, not the actual reflection type. It didn't occur to me the extra semantics my name might have introduced though, so I see your issue. Perhaps a name like `ObjectEventGetNearbyReflectionType` captures its behaviour better? Or if you think the original name was fine as-is, I can revert that change.